components/gc-featured-link/reports/a11y-2-en.html

```diff
@@ -0,0 +1,16 @@
+---
+{
+	"title": "Accessibility assessment #1 - Featured link",
+	"language": "en",
+	"description": "Evaluation of Success Criteria at Level A and Level AA of WCAG 2.1",
+	"tag": "gc-featured-link",
+	"parentdir": "gc-featured-link",
+	"parentPage": "Featured link",
+	"parentPageURL": "gc-featured-link",
```

Comment:
We don't need all those following front-matter variable for a page in GCWeb because they are not use.
```suggestion
```

Reply:
Removed

Reply:
It is back, can you remove it.

Reply:
done